vcd/resource_vcd_nsxt_network_dhcp_test.go

```diff
@@ -35,6 +35,22 @@ func TestAccVcdOpenApiDhcpNsxtRouted(t *testing.T) {
 		return
 	}
 
+	vcdClient := createTemporaryVCDConnection(true)
+	vcdVersionIsLowerThan1031 := func() (bool, error) {
+		if vcdClient != nil && vcdClient.Client.APIVCDMaxVersionIs(">= 36.1") {
+			return false, nil
+		}
+		return true, nil
+	}
+
+	// This case is specific for VCD 10.3.1 onwards since dns servers are not present in previous versions
+	var configText2 string
+	if vcdClient != nil && vcdClient.Client.APIVCDMaxVersionIs(">= 36.1") {
```

Comment:
this check isn't needed I think, as vcdVersionIsLowerThan1031 will manage the skip part

Reply:
I did that check because @Didainius suggested to avoid calling `templateFill` function as per his comment https://github.com/vmware/terraform-provider-vcd/pull/830#discussion_r855130417